src/pages/Explore.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import * as Location from 'expo-location'
+import haversine from 'haversine'
+import { getParks } from '../utils/data'
+import { distanceFilter, getClosestParksByID } from '../utils/helpers'
+import CarouselCard from '../components/CarouselCard'
+import CarouselHeader from '../components/CarouselHeader'
+import {
+  ExplorePage,
+  ExploreScrollView,
+  ExploreHeader,
+  ExploreSection,
+  ParkCardContainer,
+} from './Explore.styles.js'
+
+const Explore = () => {
+  const [parks, setParks] = React.useState([])
+  const [location, setLocation] = React.useState(null)
+
+  React.useEffect(() => {
+    let subscription
+    ;(async () => {
+      let { status } = await Location.requestPermissionsAsync()
+      if (status !== 'granted') return
+
+      const location = await Location.getCurrentPositionAsync({})
+      setLocation({
+        latitude: location.coords.latitude,
+        longitude: location.coords.longitude,
+      })
+
+      subscription = await Location.watchPositionAsync({}, (location) =>
```

Comment:
I assume this watched the user's location and updates when they move? I don't think the app would/should do this. I think we might instead want to have some sort of refresh option. Updating on the fly might create odd UX behaviours for these filtered lists. My initial thoughts would be to have a pull down to refresh, or simply a refresh button somewhere.

I think this is fine for this PR though, and if we want to change it, we can address it later.

Reply:
Hmm.. good call. I didn't think about how it would impact the filter. You are right though, it could have strange UX behaviours. I will create another issue to address this later so we don't delay pushing to Test Flight. Thanks for pointing this out!